src/main/java/at/hannibal2/skyhanni/config/features/misc/LeaveJoinMsgsConfig.java

```diff
@@ -0,0 +1,57 @@
+package at.hannibal2.skyhanni.config.features.misc;
+
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LeaveJoinMsgsConfig {
+
+    @Expose
+    @ConfigOption(
+        name = "Enabled",
+        desc = "Show a message in chat when a player leaves/joins your Private Island/Garden.\n§c§lMay be buggy in large hubs.")
+    @ConfigEditorBoolean
+    public boolean enabled = false;
```

Comment:
This should have the feature toggle notation

Reply:
wdym

Reply:
The @ FeatureToggle thing